New example to add a tag to the log.md (#33900)
DataDog/documentation: content/en/observability_pipelines/guide/get_started_with_the_custom_processor.md

### Example to convert an attribute to a tag

#### Input

Sample log containing the `ddtags` array with Datadog tags. In this example, we are trying to add the `service` field as a tag.

```json
{
    "timestamp": "2025-005-27T05:26:18.205Z",
    "status": "info",
    "service": "chaos-engineering",
    "ddsource": "python",
    "hostname": "gke-prod-node-abc123.internal",
    "message": "2025-05-27 05:26:17,609 -- Sending request to rails: checkout_v2",
    "source_type": "datadog_agent",
    "ddtags": [
        "env:prod",
        "team:sre",
        "version:1.0.0",
        "pod_name:load-generator-main-abcde"
    ]
}
```

#### Custom function to convert the `service` attribute to a tag

```yaml
# First, check if the attribute 'ddtags' exists, you can replace 'ddtags' with the name of any array
if !exists(.ddtags) {
    .ddtags = []
}

> # This checks if 'service' exists, then adds the templatized value of service as a tag. Also, converts to type string

Reviewer: Is this correct? I'm not sure what's getting converted to a string.

```suggestion
# This checks if 'service' exists, then adds the templatized value of service as a tag. Also, converts the service value to a string
```

if exists(.service) {
  .ddtags = push(array!(.ddtags), "service:" + to_string!({{.service}}) )
}

```

#### Output

```json
{
    "timestamp": "2025-005-27T05:26:18.205Z",
    "status": "info",
    "service": "chaos-engineering",
    "ddsource": "python",
    "hostname": "gke-prod-node-abc123.internal",
    "message": "2025-05-27 05:26:17,609 -- Sending request to rails: checkout_v2",
    "source_type": "datadog_agent",
    "ddtags": [
        "env:prod",
        "team:sre",
        "version:1.0.0",
        "pod_name:load-generator-main-abcde"
    ]
}
```
### Example to create a new tag and add it

#### Input

> Sample log containing the `ddtags` array with Datadog tags. In this example, we are creating a new tag called `"system:service-mesh"` and appending it to the array.

Reviewer: ```suggestion
. In this example, the sample log contains the `ddtags` array and you want to create a tag called `"system:service-mesh"` and append it to the array.
```
